packages/thumbprint-react/components/Image/index.tsx

```diff
@@ -19,23 +18,87 @@ import styles from './index.module.scss';
 // 5. The image onLoad and onError events remove padding-top placholder and sets opacity to 1.
 // --------------------------------------------------------------------------------------------
 
-const Image = forwardRef((props, outerRef) => {
+type ImageSource = {
+    type: 'image/webp' | 'image/jpeg' | 'image/png' | 'image/gif';
+    srcSet: string;
+};
+
+interface ImagePropTypes {
+    /**
+     * If `sources` is provided, this image will be loaded by search engines and lazy-loaded for
+     * users on browsers that don't support responsive images. If `sources` is not provided, this
+     * image will be lazy-loaded.
+     */
+    src: string;
+    /**
+     * Allows the browser to choose the best file format and image size based on the device screen
+     * density and the width of the rendered image.
+     */
+    sources: ImageSource[];
+    alt?: string;
+    /**
+     * Crops the image at the provided height. The `objectFit` and `objectPosition` props can be
+     * used to control how the image is cropped.
+     */
+    height?: string;
+    /**
+     * Creates a [placeholder box](https://css-tricks.com/aspect-ratio-boxes/) for the image.
+     * The placeholder prevents the browser scroll from jumping when the image is lazy-loaded.
+     */
+    containerAspectRatio?: number;
+    /**
+     * Disables lazy-loading and overrides the default calculation of the `sizes` attribute.
+     * Primarily for important images in a server-side rendered environment that must be
+     * loaded before JavaScript is parsed and executed on the client. The value gets used
+     * as the `sizes` attribute. [See allowable values](https://mzl.la/2Hh6neO).
+     */
+    forceEarlyRender?: boolean | string;
```

Comment:
We could be more explicit here about the type of string allowed.

```suggestion
    forceEarlyRender?: boolean | React.ImgHTMLAttributes<HTMLImageElement>["sizes"];
```

Reply:
done